utils/pagerduty_api.py

```diff
@@ -51,7 +63,7 @@ def get_escalation_policy_users(self, escalation_policy_id, now):
                         self.get_schedule_users(target['id'], now)
                     users.extend(schedule_users)
                 elif target_type == 'user_reference':
-                    users.append(target['summary'])
+                    users.append(self.get_user(target['id']))
```

Comment:
What happens if you get `None` here?

Reply:
i changed the `return None` to raise an exception to prevent this.